src/SFA.DAS.Campaign.Web/Views/Home/Countries.cshtml

```diff
@@ -3,7 +3,7 @@
 @using SFA.DAS.Campaign.Web.ViewComponents.HeroHeading
 @{
     Layout = "_LayoutContent";
-    ViewBag.Title = "WHERE ARE YOU BASED";
+    ViewBag.Title = "WHERE ARE YOU BASED?";
```

Comment:
Can we change this to sentence case? i.e 'Where are you based?' Its better for screen readers